.gitignore

```diff
@@ -31,4 +31,7 @@ node_modules
 
 # Dist folder
 dist/*
-!dist/.gitkeep
\ No newline at end of file
+!dist/.gitkeep
+
+# Vercel
+.vercel
```

Comment:
When using the [Vercel CLI](https://vercel.com/docs/cli) in this project, a `.vercel` directory is created with some generated files that shouldn't be committed.